portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -4,40 +4,41 @@
 
 /** Represents a comment with related details. */
 public final class Comment implements EntityConvertible {
+    private final long id;
     private final String commenter;
     private final String content;
     private final long timestamp;
 
-    public Comment(String commenter, String content, long timestamp) {
+    public Comment(long id, String commenter, String content, long timestamp) {
+        this.id = id;
         this.commenter = commenter;
         this.content = content;
         this.timestamp = timestamp;
     }
 
-    static class Constants {
-        final static String KIND = "Comment";
-        final static String COMMENTER = "commenter";
-        final static String CONTENT = "content";
-        final static String TIMESTAMP = "timestamp";
-    }
+    private static final String KIND = "Comment";
```

Comment:
Maybe move these constant to top of class.

Reply:
Fixed